packages/pwa/app/components/_app/index.jsx

```diff
@@ -168,6 +176,25 @@ const App = (props) => {
                             href={getAssetUrl('static/img/global/apple-touch-icon.png')}
                         />
                         <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
+
+                        {/* Urls for all localized versions of this page (including current page)
+                        For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
+                        {SUPPORTED_LOCALES.map((locale) => (
+                            <link
+                                rel="alternate"
+                                hrefLang={locale.toLowerCase()}
+                                href={`${appOrigin}${buildUrlLocale(locale)}`}
+                                key={locale}
+                            />
+                        ))}
```

Comment:
Yes, the current page's url should be in this list too. As per Google: `Each language version must list itself as well as all other language versions`

Reply:
Are we supposed to do this for each page? or just the home page? 🤔 . My impression is that we want to do on each page since we have them localized

Reply:
Sorry my bad, I mean when we navigate from one page to another, I can only see the `x-default` on the new page. is that the expected behavior?

Reply:
@alexvuong Oh the other hreflang links are in there somewhere. Due to re-rendering, they don't appear in order anymore. Try searching for them with Cmd + F .